app/services/playlist_creator.rb

```diff
@@ -0,0 +1,34 @@
+class PlaylistCreator
+
+  def initialize(genre, energy, danceability, playlist)
+    @genre = genre
+    @energy = energy
+    @danceability = danceability
+    @playlist = playlist
+  end
```

Comment:
Niveau interface tu as juste besoin de playlist:

```suggestion
  def initialize(playlist)
    @playlist = playlist
  end
```

Et ensuite les 3 autres variables:

```ruby
def genre
  @playlist.genre
end
```